tests/test_font.py

```diff
@@ -643,3 +658,47 @@ def test_FontMetrics_hasStrikeoutThickness(fontmetrics):
 def test_FontMetrics_hasStrikeoutPosition(fontmetrics):
     position = 0.
     assert isinstance(fontmetrics.hasStrikeoutPosition(position), bool)
+
+
+@pytest.fixture
+def color_emoji_run():
+    if sys.platform.startswith("linux"):
+        if (os.getenv("GITHUB_ACTION") == True):
```

Comment:
Maybe check the file with `os.exists()` instead of relying on the CI runner environment?

Reply:
That's a good idea. I'll do that. Strangely the font is available on Ubuntu but not accessible via fontconfig. (And at a different place on fedora)

Reply:
Done that.